IPE-1510 - details about IP allowlist exemptions (#2529)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/users-teams-organizations/ip-allowlists.mdx

IP allowlists define sets of IP address ranges in CIDR notation that are permitted to access specific HCP Terraform resources. When you configure an IP allowlist, HCP Terraform checks the source IP address of incoming requests and only allows access if the IP address matches one of the ranges in the allowlist.

> ### Exceptions

Reviewer: Good location for the Exceptions section, flows nicely and puts all the important info up front.  Might even help with feature adoption because they realize they don't need to add IP lists for GitHub.

[VCSs connected to the platform](/terraform/cloud-docs/vcs) and [organization owners](/terraform/cloud-docs/users-teams-organizations/permissions/organization#organization-owners) can always access platform resources, even when accessing the platform from an IP that isn't on the allowlist. Follow secure best practices for systems and users that may access HCP Terraform.   

## Requirements

> To allow mobile team members to access resources, they must connect from an IP address on the allowlist. You can use a VPN or another networking solution to provide static IP addresses for mobile team members. Refer to you system administrator for more information.

Reviewer: ```suggestion
To access resources, team members must connect from an IP address on the allowlist. You can use a VPN or another networking solution to provide static IP addresses for mobile team members. Refer to your system administrator for more information.
```

Mostly wanted to make the first sentence broader, so readers didn't think this only applied to mobile users somehow, or go looking for non-mobile info.
